script/DeployBase.sol

```diff
@@ -2,11 +2,8 @@
 
 pragma solidity 0.8.23;
 
-import { Script, console2 } from "../lib/forge-std/src/Script.sol";
```

Comment:
why it was renamed?

Reply:
Because of the following comment in the Integration tests:
```
// NOTE: Using `DeployBase` as a contract instead of a script, means that the deployer is `_deploy` itself.
```